WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ShippingLabelPaperSizeOptionsViewController.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+import UIKit
+
+/// Displays a grid view of all available paper size options for printing a shipping label.
+final class ShippingLabelPaperSizeOptionsViewController: UIHostingController<ShippingLabelPaperSizeOptionsView> {
+    private let onCloseButtonTapped: () -> Void
+
+    /// - Parameter onCloseButtonTapped: Called when the user taps on the close button in the navigation bar.
+    init(onCloseButtonTapped: @escaping () -> Void) {
+        self.onCloseButtonTapped = onCloseButtonTapped
+        super.init(rootView: ShippingLabelPaperSizeOptionsView(paperSizeOptions: [.legal, .letter, .label]))
+        configureNavigationBar()
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension ShippingLabelPaperSizeOptionsViewController {
+    func configureNavigationBar() {
+        navigationItem.title = Localization.navigationBarTitle
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: .closeButton, style: .plain, target: self, action: #selector(closeButtonTapped))
```

Comment:
Should we use `addCloseNavigationBarButton` helper ?

Reply:
Makes sense! Updated in https://github.com/woocommerce/woocommerce-ios/pull/3322/commits/7e67eccacf8ef02259874996d0886ed08259cec4 (also updated the testing code in PR description)